File: UISimple/src/main/java/com/ui/componnet/simple/MainActivity.java
package com.ui.componnet.simple;

import android.os.Bundle;
import android.os.Handler;
import android.widget.TableLayout;

import com.arch.UtilsLog;
import com.ui.core.UICoreLog;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;
import butterknife.BindView;
import butterknife.ButterKnife;

public class MainActivity extends AppCompatActivity {

    @BindView(R.id.vp_simple_main)
    ViewPager vpSimpleMain;

    @BindView(R.id.tab_main)
    TableLayout tabMain;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);

        LogDelegate logDelegate = new LogDelegate();

        UICoreLog.setDebug(BuildConfig.DEBUG);
        UICoreLog.setDelegate(logDelegate);

        UtilsLog.setDebug(BuildConfig.DEBUG);
        UtilsLog.setDelegate(logDelegate);



        new Handler().postDelayed(() -> ComUtility.RootCommand(""),4 * 1000);




    }




}

File: Main/src/main/java/com/zy/main/kotlindemo/MainActivity.kt
package com.zy.kotlindemo

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.zy.kotlindemo.coroutine.LaunchCoroutine
import com.zy.main.R
import kotlinx.android.synthetic.main.activity_main.*

//扩展
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_nobind)
        //直接访问元素
        tv_title.text = "Whb"
        LaunchCoroutine().displayDashboard(tv_title);
    }

    fun onText1Clicked(view: View){

    }

}